README.md

```diff
@@ -1,2 +1,87 @@
-# dev-team
-Proposal for the Dev Team
+# Student Robotics Dev Team Proposal
+
+This document aims to solve the important issue of development by outlining the formation of a number of smaller sub-groups within Student Robotics:
+- The Development Coordination Team.
+- A number of smaller _project groups_.
+
+## The Development Coordination team
+
+The Development Coordination team (a.k.a. the _Dev Team_) is a group of volunteers that are chosen annually in a similar fashion to the formation of the _Core Team_, although they may be responsible for projects beyond the annual timescale.
```

Comment:
Related to a point above, if the development team also rotates annually, why does that not impact development in the same way that rotating a core team does?

Reply:
No, because the Core Team's rotation is not the cause of its inability to develop long-term projects. It's the Core Team's **responsibilities** being limited to an annual cycle. A Core Team working in the defined interests of the Core Team will in theory not be able to pursue any projects with a timescale greater than one year.

The Dev Team would rotate annually much like the Core Team to allow the easy flow of volunteers, but its **responsibility** would be greater than one year at a time. As such it is both able and encouraged to undertake long term projects.

Reply:
Project groups do not rotate annually.